src/components/gridlines.ts

```diff
@@ -42,6 +64,36 @@ export class Gridlines extends Component {
     }
   }
 
+  public betweenX(_betweenX: boolean): this;
```

Comment:
oh as in re-interpret the "main" gridlines to be between the ticks? Hmm I see, that makes sense for Category axis I think but maybe not linear. I think we can all agree on "gridlines do between ticks for Category", maybe we only implement that for now and leave the trickiness of simple averaging for later